Document/0x05a-Platform-Overview.md

```diff
@@ -527,7 +527,7 @@ When an application is installed on the Android device, the Package Manager ensu
 
 #### APK Signing Schemes
 
-Android supports two application signing schemes. Starting with Android 7.0, APKs can be verified with the APK Signature Scheme v2 (v2 scheme) or JAR signing (v1 scheme). For backwards compatibility, APKs signed with the v2 signature format can be installed on older Android devices as long as the former are also v1-signed. [Older platforms ignore v2 signatures and verify v1 signatures only](https://source.android.com/security/apksigning/ "APK Signing ").
+Android supports three application signing schemes. Starting with Android 9.0, APKs can be verified with APK Signature Scheme v3 (v3 scheme), APK Signature Scheme v2 (v2 scheme) or JAR signing (v1 scheme). For Android 7.0 and above, APKs can be verified with the APK Signature Scheme v2 (v2 scheme) or JAR signing (v1 scheme). For backwards compability, APKs signed with a given signing scheme can be installed on older Android devices as long as the former are signed with prior scheme verions too. [Older platforms ignore v2 signatures and verify v1 signatures only](https://source.android.com/security/apksigning/ "APK Signing ").

```

Comment:
The `For backwards compability, APKs signed with a given signing scheme can be installed on older Android devices as long as the former are signed with prior scheme verions too.` could be rephrased to `For backwards compatibility, an APK can be signed with multiple signature schemes in order to make the app run on both newer and older SDK versions.`